### PR TITLE
Added calculator/math_parser.py full code

### PR DESCRIPTION
**Pull request by izanbf1803**
Working implementation of a "calculator" (math expression parser) using the Shunting-yard algorithm.

### CODE REVIEW

File: calculator/math_parser.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+def isOperator(token):
+	"""
+	Check if token it's a operator
+
+	token Char: Token
+	"""
+	return token in __operators__
+
+def higherPriority(op1, op2):
```
Comment (keon) [inline calculator/math_parser.py]: snake case

File: calculator/math_parser.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+__operators__ = "+-/*"
+__parenthesis__ = "()"
+__priority__ = {
+	'+': 0,
+	'-': 0,
+	'*': 1,
+	'/': 1,
+}
+
+def isOperator(token):
```
Comment (keon) [inline calculator/math_parser.py]: Please convert it to snake case

File: calculator/math_parser.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+	return result
+
+def evaluate(expression):
+	"""
+	Calculate result of expression
+
+	expression String: The expression
+	type Type (optional): Number type [int, float]
+	"""
+	opStack  = deque() # operator stack
```
Comment (keon) [inline calculator/math_parser.py]: snake case for the values, too

File: calculator/math_parser.py [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+		|	Evaluation result: 1189.4808510638297
+-------------------------------------------------------------------------------------------------
+"""
+
+from collections import deque
+
+__operators__ = "+-/*"
+__parenthesis__ = "()"
+__priority__ = {
+	'+': 0,
```
Comment (keon) [inline calculator/math_parser.py]: 4 spaces instead of tab

**Comment by izanbf1803** [comment]
Requested changes applied.

**Comment by keon** [comment]
awesome thanks

**Comment by izanbf1803** [comment]
About the question of move related stuff to other repo, i think this is algorithmic stuff. It uses Dijkstra's Shunting-yard algorithm.

**Pull request merged**